image-viewer.py

```diff
@@ -55,5 +58,82 @@ def render(self):
 
         self.root.bind("<KeyPress>", self.onKeyPress)
         self.root.mainloop()
+        
+    # Menu creation
+    def create_menu(self):
+        menubar = Menu(self.root) # Create the main Menu bar
+        self.root.config(menu=menubar) # Attach the Menu bar to the main root window
+        
+        file_menu = Menu(menubar, tearoff=False) # Create the 'File' sub menu
```

Comment:
I had no idea what `tearoff` was prior to this, but I must say that it definitely looks better as False 😆